python/eo_acquisition.py

```diff
@@ -150,7 +151,7 @@ def _check_subsystems(self):
         """
         Check that the required subsystems are present.
         """
-        required = 'ts8 pd mono'.split()
+        required = 'ts8 pd mono ts8dac0 ts8dac1 ts8dac2'.split()
```

Comment:
Just to confirm:  Based on the usage of the `ccd_clear` function below, these three new subsystems are indeed required.  Is that right?

Reply:
They are needed only for e2v in uni-polar clocking mode but the required occurs before the test for ccd type and mode.
So I don't see any choice but to say yes.  Open to suggestions.

Reply:
ok, that's fine.  The only alternative would be to leave it up to the operator to know that these have to be specified when running with e2v sensors.